test/e2e/redundancy/gateway_failover.go

```diff
@@ -25,6 +25,7 @@ import (
 	. "github.com/onsi/gomega"
 	"github.com/submariner-io/shipyard/test/e2e/framework"
 	"github.com/submariner-io/shipyard/test/e2e/tcp"
+	"github.com/submariner-io/submariner/test/e2e/dataplane"
```

Comment:
We don't want the `redundancy` package referencing `dataplane` as that will cause the ginkgo test specs to get pulled in as well. Perhaps we should put the new functions in shipyard as well? Will we need `tcp.RunConnectivityTest` anymore?

Reply:
Done